index.js

```diff
@@ -61,10 +61,10 @@ class instance extends instance_skel {
 		];
 
 		this.CHOICES_OUTPUTFORMAT = [
-			{ id: '50i', label: '50i',  preset: '50i' },
-			{ id: '50p', label: '50p',  preset: '50p' },
-			{ id: '60i', label: '60i',  preset: '60i' },
-			{ id: '60p', label: '60p',  preset: '60p' }
+			{ id: '1080i50', label: '1080i50',  preset: '1080i50' },
+                        { id: '1080i5994', label: '1080i5994',  preset: '1080i5994' },
```

Comment:
```suggestion
                        { id: '1080i5994', label: '1080i59.94',  preset: '1080i59.94' },
```

Reply:
Now covered in https://github.com/bitfocus/companion-module-bmd-multiview4/pull/6#discussion_r1527170486